src/WalkingModule/app/robots/iCubGenova04/dcm_walking/joypad_control/zmpControllerParams.ini

```diff
@@ -4,7 +4,7 @@ useGainScheduling      1
 smoothingTime          0.1
 
 # if the gain scheduling is off only k*_walking parameters are used
-kZMP_walking           3.5
```

Comment:
Please avoid changing this file since we are using this gains for public demos. If you think there is a problem please open an issue and an associate PR only for this

Reply:
Done!